client/src/containers/ClubProfile.js

```diff
@@ -43,12 +47,24 @@ export const ClubProfile = () => {
             />
     )))};
 
+    const ConditionalAccountTag = () => { return (firstLogIn === true) ? 
+        ( <AccountTag/> ) : 
+        (null) 
+    };
+
+    const ConditionalClubProfile = () => { return (isLoggedin === true) ? 
+        ( <YourClubProfile/> ) : 
+        (null) 
+    };
+
     if (width < 541){ // mobile view
         return (
             <>
                 {!isLoading && (
                     <Wrapper>
                         <Navbar/>
```

Comment:
Here is where you could use a conditional function as you did for the account tag and the other component to render the navbar based on `isLoggedIn`. This way you don't have two separate variables for `isLoggedIn` in the navbar file and here, just use the same variable to make all of the decisions